tests/features/local_translations.feature

```diff
@@ -41,8 +51,25 @@ Feature: Local translations
     Then I should see "Bulgarian title"
     And I should see "Bulgarian body"
 
+    # Create a new translation for the same language.
+    When I click "Translate"
+    And I click "Translate locally" in the "Bulgarian" row
+    Then the translation form element for the "Title" field should contain "Bulgarian title"
+    And the translation form element for the "Body" field should contain "Bulgarian body"
+    And the translation form element for the "Link Uri (1)" field should contain "https://example.com/bg"
+    And the translation form element for the "Link Title (1)" field should contain "Example BG"
+    And the translation form element for the "Link Uri (2)" field should contain "/bg/node"
+    And the translation form element for the "Link Title (2)" field should contain "Node BG"
+
+    When I fill in the translation form element for the "Title" field with "Bulgarian title 2"
+    And I press "Save and complete translation"
+    Then I should see "Translations of Bulgarian title 2"
+    When I click "Bulgarian title 2" in the "Bulgarian" row
+    Then I should see "Bulgarian title 2"
+    And I should see "Bulgarian body"
+
   Scenario: Preview translations
-    Given a translatable node with the "My title" title and "My body" body
+    Given a translatable node with the "My title" title and "My body" body and multiple links
```

Comment:
Related to the comment above, here we add links when they are not needed? If we had separate steps it would be clearer in my opinion